pkg/s3/bucket.go

```diff
@@ -81,6 +84,7 @@ func EncryptBucket(s3Client *s3.S3, bucketName string) error {
 	return err
 }
 
+// BlockBucketPublicAccess blocks public access to the bucket's contents.
```

Comment:
```suggestion
// BlockBucketPublicAccess blocks public access to the buckets contents.
```

Reply:
Are you sure about this one? `bucket's` should  be possessive (apostrophe), since we're referring to the bucket's contents.

Reply:
Sorry, you're right!